components/scream/src/share/field/field.hpp

```diff
@@ -16,18 +16,17 @@ namespace scream
 
 // A field should be composed of metadata info (the header) and a pointer to the view
 
-template<typename ScalarType, typename MemSpace, typename MemManagement>
+template<typename ScalarType, typename D=DefaultDevice>
```

Comment:
I disagree (not strongly). I think the default device is a reasonable default since that's what the developer will want in the vast majority of cases.

Reply:
@bartgol , sure. I was also thinking that the defaults allow an outside chance of an error as you described. I suppose that probably outweighs the benefits of reduced typing. Let me change it.